source/module_io/cube_io.h

```diff
@@ -8,45 +8,91 @@
 
 namespace ModuleIO
 {
-bool read_cube(
+extern bool read_cube(
```

Comment:
why you add 'extern' before read_cube?

Reply:
@mohanchen the extern prefix has been deleted